replace/modernized/07-chapter-7.adoc

```diff
@@ -688,7 +693,7 @@ where no unclean person can come, Eph. 5:5, Isa. 38:18,
 neither can death nor the grave praise the Lord,
 nor they that go down to the pit hope for his Truth.
 
-Objection.
```

Comment:
The objections and query's in this chapter all need to be [.discourse-part] too 👍🏻